server/src/main/java/io/spine/server/entity/storage/Column.java

```diff
@@ -76,16 +79,23 @@
  *      {@code
  *         --UserAggregate.java--
  *
+ *         // only methods annotated with @javax.persistence.Column may be considered Columns
+ *         public Department getDepartment() { ... }
+ *
  *         // non-public methods may not represent Columns
+ *         \@Column
```

Comment:
Please check if this form is interpreted correctly.

Reply:
Javadocs will contain backslashes, but there are no better options due to the [bug](https://bugs.openjdk.java.net/browse/JDK-8130754).